nym-validator-rewarder/src/config/mod.rs

```diff
@@ -129,7 +129,10 @@ impl Config {
             database_path: self.storage_paths.nyxd_scraper.clone(),
             pruning_options: self.nyxd_scraper.pruning,
             store_precommits: self.nyxd_scraper.store_precommits,
-            start_block_height: None,
+            start_block: StartingBlockOpts {
+                start_block_height: None,
+                use_best_effort_start_height: true,
```

Comment:
@jstuczyn hope this doesn't mess with the rewarder logic 👀

Reply:
I don't think it should because the rewarded has already been started so those settings wouldn't apply. we just need to make sure the config correctly deserialises